roscala/src/main/scala/coop/rchain/rosette/VirtualMachine.scala

```diff
@@ -257,7 +262,7 @@ object VirtualMachine {
     currentState
   }
 
-  def runFlags(state: VMState): VMState = {
+  def runFlags: VMTransition = modify { state =>
     var mState = state
```

Comment:
I focused on the `execute` function in this PR since this is what we touch on a daily basis. You are right though. However would prefer to do that in another PR.

Reply:
👍 maybe just add TODO ? that's a low-hanging-fruit, maybe from community or sth